File: toolbardemo/src/main/java/com/easyliu/demo/toolbardemo/MainActivity.java
package com.easyliu.demo.toolbardemo;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.support.v7.widget.Toolbar;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        Toolbar myToolbar = (Toolbar) findViewById(R.id.my_toolbar);
        setSupportActionBar(myToolbar);
    }
}

File: glide_demo/src/main/java/com/easyliu/demo/glide_demo/MainActivity.kt
package com.easyliu.demo.glide_demo

import android.graphics.Color
import android.graphics.drawable.ColorDrawable
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.widget.ImageView
import com.bumptech.glide.Glide

class MainActivity : AppCompatActivity() {
    private val img by lazy {
        findViewById<ImageView>(R.id.iv_img)
    }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        Glide.with(this)
                .load("http://i.gtimg.cn/qqlive/images/20191209/i1575881814_1.jpg")
                .centerCrop()
                .circleCrop()
                .placeholder(ColorDrawable(Color.RED))
                .into(img);
    }
}